crates/sui-adapter-transactional-tests/tests/sui/move_call_incorrect_function.exp

```diff
@@ -6,7 +6,7 @@ written: object(103)
 
 task 2 'run'. lines 16-18:
 Error: Transaction Effects Status: Move Bytecode Verification Error. Please run the Bytecode Verifier for more information.
-Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: VMVerificationOrDeserializationError, source: Some(VMError { major_status: LINKER_ERROR, sub_status: None, message: Some("Cannot find ModuleId { address: _, name: Identifier(\"M\") } in data cache"), exec_state: None, location: Undefined, indices: [], offsets: [] }), command: Some(0) } }
+Execution Error: ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: VMVerificationOrDeserializationError, source: Some(VMError { major_status: LINKER_ERROR, sub_status: None, message: Some("Cannot find link context _ in store"), exec_state: None, location: Undefined, indices: [], offsets: [] }), command: Some(0) } }
```

Comment:
@tnowacki, re: backward compatibility -- am I right in thinking/hoping that this kind of change in errors (just to the message) is ok? (not actually part of the full effects, and therefore won't affect the digest)?

Reply:
(Confirmed that this is the case offline).